example/vanilla-k8s-block-driver/vsphere.conf

```diff
@@ -7,3 +7,9 @@ user = "vcenter username"
 password = "vcenter password"
 port = "443"
 datacenters = "list of comma seperated datacenters where node VMs are present"
+
+# FeatureStatesConfig holds the details about feature states configmap.
```

Comment:
Can we also mention that the default config-map name is `csi-feature-states` and default namespace is `kube-system` if not specified?

only if feature state config-map is in the different namespace we need to specifiy it in the `vsphere.conf` file.

Reply:
Done